script.js

```diff
@@ -0,0 +1,41 @@
+const forms = document.getElementById("form");
+const inputs = document.getElementById("input");
+const btn = document.getElementById("button");
+const tables = document.getElementById("table");
+
+// const tr = document.createElement("tr");
+// document.getElementById.appendChild(tr);
+let result = 1;
+
+forms.addEventListener("submit", (e) => {
+  e.preventDefault();
+});
```

Comment:
Isto é apenas preferências, mas quando as funções são assim pequenas eu gosto de meter tudo na mesma linha:

```suggestion
forms.addEventListener("submit", e => e.preventDefault());
```

E se quiseres até podes fazer assim para simplificar:
```suggestion
document.getElementById("form").addEventListener("submit", e => e.preventDefault());
```